### PR TITLE
Update PowerFlex replication instruction

### PR DESCRIPTION
**Pull request by falfaroc**
# Description
Address multiple suggestions for improvement in the PowerFlex Replication documentation.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/817 |
| https://github.com/dell/csm/issues/818 |

# Checklist:

- [x] Have you run grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/replication/deployment/powerflex.md [hunk 1]
```diff
@@ -257,6 +259,18 @@ replication enabled Storage Classes. The CSI PowerFlex driver will create a
...
+To delete a replication group from a cluster, the group must be empty. This
+means that all associated replication pairs must have been deleted/removed.
+
+This command will delete the empty replication group from the cluster and the
+associated replication consistency group from the PowerFlex array.
+
+```shell
+  kubectl delete rg <rg-id>
+```
+
```
Comment (anandrajak1) [inline content/docs/replication/deployment/powerflex.md:273]: Can this added to tools.md, Its not specific to powerfkex
@santhoshatdell - Let me know your comments ?

File: content/docs/replication/deployment/powerflex.md [hunk 2]
```diff
@@ -36,15 +36,16 @@ Run the following commands to verify that everything is installed correctly:
   Pods should be `READY` and `RUNNING`
 * Check that the controller config map is properly populated
     ```shell
-
```
Comment (mgandharva) [inline content/docs/replication/deployment/powerflex.md:39]: why is this line removed?

File: content/docs/replication/deployment/powerflex.md [hunk 3]
```diff
@@ -36,15 +36,16 @@ Run the following commands to verify that everything is installed correctly:
...
     ```
   `data` field should be properly populated with cluster-id of your choosing
   and, if using multi-cluster installation, your `targets:` parameter should be
   populated by a list of target cluster IDs.
 
 
-If you don't have something installed or something is out-of-place, please refer to installation instructions [here](../installation).
+If you don't have something installed or something is out-of-place, please refer
+to installation instructions in [installation-repctl](../install-repctl) or
+[installation](../installation).
```
Comment (santhoshatdell) [inline content/docs/replication/deployment/powerflex.md:48]: Could you revert this change? We updated the installation part of docs recently,

File: content/docs/replication/deployment/powerflex.md (hunk 1 above)
Comment (santhoshatdell) [reply content/docs/replication/deployment/powerflex.md:273]: I agree. Delete RG is not specific to PowerFlex. It can be moved to a common place.
Tools.md seems to be repctl specific. How about replication-actions page or some other platform generic page ?

**Review by rsedlock1958 — APPROVED** [review]

File: content/docs/replication/deployment/powerflex.md (hunk 2 above)
Comment (falfaroc) [reply content/docs/replication/deployment/powerflex.md:39]: For consistency and to avoid a leading line in the codeblock.

File: content/docs/replication/deployment/powerflex.md (hunk 1 above)
Comment (falfaroc) [reply content/docs/replication/deployment/powerflex.md]: Moved to `replication-actions` since `tools` is more focused on `repctl` actions.

File: content/docs/replication/replication-actions.md [hunk 4]
```diff
@@ -61,3 +61,14 @@ While performing site-specific actions, please consult each driver's documentati
 
 For a brief guide on using actions for various DR workflows, please refer to this [document](../disaster-recovery). 
 
+### Deleting Replication Groups
+
+To delete a replication group from a cluster, the group must be empty. This
+means that all associated replication pairs must have been deleted/removed.
+
+This command will delete the empty replication group from the cluster and the
+associated replication consistency group from the PowerFlex array.
```
Comment (santhoshatdell) [inline content/docs/replication/replication-actions.md:70]: How about 'associated **replication group** from the **backend** array' ?

**Review by santhoshatdell — APPROVED** [review]

File: content/docs/replication/replication-actions.md (hunk 4 above)
Comment (falfaroc) [reply content/docs/replication/replication-actions.md]: Good catch, typo from copying it over.

**Review by santhoshatdell — APPROVED** [review]

**Comment by falfaroc** [comment]
@rsedlock1958 Can you look over and reapprove, pending on a code owner?

**Comment by rsedlock1958** [comment]
Yes. Please re-send me the link

**Review by rajkumar-palani — APPROVED** [review]
lgtm
